### PR TITLE
[ADD] membership migration

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: addons/membership/migrations/8.0.0.1/post-migrate.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@openupgrade.migrate()
+def migrate(cr, version):
+    cr.execute(
+        '''update product_template set
+        membership = p.membership,
+        membership_date_from = p.membership_date_from,
+        membership_date_to = p.membership_date_to
+        from product_product p
+        where p.product_tmpl_id = product_template.id''')
+    cr.execute(
```
Comment (pedrobaeza) [inline addons/membership/migrations/8.0.0.1/post-migrate.py]: Better to let this job be done by database_cleanup?

**Comment by StefanRijnhart** [comment]
:+1:, I prefer like @pedrobaeza not to delete data in scripts. However, risk of dataloss in this case is nihil IMHO.

**Comment by hbrunn** [comment]
@pedrobaeza @StefanRijnhart agreed and removed

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**